cmd/zpool/zpool_main.c

```diff
@@ -1556,8 +1582,13 @@ print_status_config(zpool_handle_t *zhp, status_cbdata_t *cb, const char *name,
 		zfs_nicenum(vs->vs_write_errors, wbuf, sizeof (wbuf));
 		zfs_nicenum(vs->vs_checksum_errors, cbuf, sizeof (cbuf));
 		(void) printf(" %5s %5s %5s", rbuf, wbuf, cbuf);
+	} else {
+		(void) printf("                  ");
 	}
 
+	if (cb->cb_kind)
+		(void) printf(" %4s", kind_mark(nv));
```

Comment:
In `status` the custom seemed to be just one space as separator.  The maximum entry is 4 bytes.